test/bdd/pkg/vdri/vdri_steps.go

```diff
@@ -117,8 +117,14 @@ func (e *Steps) createDIDBloc(url, keyType, signatureSuite string) error {
 }
 
 func (e *Steps) resolveCreatedDID(url, keyType, signatureSuite string) error {
+	genesisFile, err := ioutil.ReadFile("fixtures/wellknown/jws/did-trustbloc/testnet.trustbloc.local.json")
+	if err != nil {
+		return err
+	}
+
 	blocVDRI := trustbloc.New(trustbloc.WithResolverURL(url), trustbloc.WithTLSConfig(e.bddContext.TLSConfig),
-		trustbloc.WithAuthToken("rw_token"), trustbloc.WithDomain("testnet.trustbloc.local"))
+		trustbloc.WithAuthToken("rw_token"), trustbloc.WithDomain("testnet.trustbloc.local"),
+		trustbloc.UseGenesisFile("testnet.trustbloc.local", "testnet.trustbloc.local", genesisFile))
```

Comment:
Do you need to add trustbloc.UseGenesisFile here https://github.com/trustbloc/trustbloc-did-method/blob/master/pkg/restapi/didmethod/operation/operations.go#L75

Reply:
we need to enable it in trustbloc-rest